www/js/con-author.js

```diff
@@ -292,36 +447,34 @@ define([
 				//to do: applying directives on PM processed object, for now just processing, yet to write apply directives
 				var returnObj = this.authorPM.process(this.currentID, "initial", newInitial, true);
 				console.log("author pm returned after evaluating initial value",returnObj);
-				//this.applyDirectives(this.authorPM.process(this.currentID, "initial", newInitial, true));
+				this.applyDirectives(returnObj);
 				
-				/* to do : status updates after model integration and writing necessary functions
 				var studentNodeID = this._model.student.getNodeIDFor(this.currentID);
 				var studNodeInitial = this._model.student.getInitial(studentNodeID);
-				if(modelType == "given"){
-					//if the model type is given , the last initial value is the new student model value
+				if(modelType == "authored"){
+					//if the model type is authored , the last initial value is the new student model value
 					//which in this case is second parameter
 					this._model.active.setInitial(studentNodeID, newInitial);
-					this.updateStatus("initial", this._model.given.getInitial(this.currentID), newInitial);
+					this.updateStatus("initial", this._model.authored.getInitial(this.currentID), newInitial);
```

Comment:
@jonwetzel @sachingrover211  I remember switching between value and initial in dragoon based on node type but Can't remember discussing anything as such in topomath. @jonwetzel let me know if this changes has to be made